web/cypress/e2e/onboarding-as-foreign-business.spec.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  randomElementFromArray,
+  setMobileViewport,
+} from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { onOnboardingPageNexusBusiness } from "@businessnjgovnavigator/cypress/support/page_objects/onboardingPageNew";
+import {
+  Industries,
+  Industry,
+  ResidentialConstructionType,
+  carServiceOptions,
+  randomInt,
+} from "@businessnjgovnavigator/shared/";
+
+const enabledIndustries = Industries.filter((element: Industry) => {
+  return element.isEnabled;
+});
+
+describe("Onboarding for all industries when starting a business [feature] [all] [group4]", () => {
```

Comment:
pebble - can we update all of the existing titles for all of the on boarding test that you've completed so that they correctly reflect the test. And this, and since you are on boarding as an foreign nexus business.